lib/bump/cli/commands/deploy.rb

```diff
@@ -6,17 +6,21 @@ module Commands
       class Deploy < Base
         desc "Create a new version"
         argument :file, required: true, desc: "Path or URL to your API documentation file. OpenAPI (2.0 to 3.0.2) and AsyncAPI (2.0) specifications are currently supported."
-        option :id, default: ENV.fetch("BUMP_ID", ""), desc: "Documentation public id"
-        option :token, default: ENV.fetch("BUMP_TOKEN", ""), desc: "Documentation private token"
+        option :id, default: ENV.fetch("BUMP_ID", ""), desc: "[DEPRECATED] [DEPRECATED] Use `documentation` option instead"
+        option :documentation, default: ENV.fetch("BUMP_ID", ""), desc: "Documentation id or slug"
```

Comment:
I'm wondering if `--doc` wouldn't be easy to use here, and more consistent with the `--hub` one (3 chars for each). WDYT?

Reply:
Yes I thinks so as well!

Reply:
👍